covid_graphs/covid_graphs/country_graph.py

```diff
@@ -219,5 +217,7 @@ def create_dashboard(data_dir: Path, server: Flask, graph_type: GraphType=GraphT
     type=str,
 )
 def show_country_plot(data_dir: Path, country_name: str):
-    country_graph = CountryGraph(data_dir=data_dir, country_name=country_name, graph_type=GraphType.Normal)
+    country_graph = CountryGraph(data_dir=data_dir,
+                                 country_name=country_name,
+                                 graph_type=GraphType.Normal)
```

Comment:
Btw. I plan to add to CI a style checker over the weekend, together with type checker and a testing framework.

Reply:
Actually I meant automatic code formatter :)